stac_fastapi/api/tests/test_api.py

```diff
@@ -13,6 +14,13 @@ class TestRouteDependencies:
     @staticmethod
     def _build_api(**overrides):
         settings = config.ApiSettings()
+
+        items_get_request_model = create_request_model(
+            "ItemCollectionURI",
+            base_model=ItemCollectionUri,
+            mixins=[TokenPaginationExtension().GET],
+        )
```

Comment:
This would need to be documented. 

Maybe we should create a `v2.0 to v3.0 migration guide`

Reply:
Sounds really good to me - quite a lot going on all in all